### PR TITLE
fix(frontend): During startup, enable text field, but disable submit …

### PR DESCRIPTION
**Pull request by Yashwanth-Chandrakumar**
…#182

Removed the disabled attribute from text box and add it to button.

The button would be disabled until the model is initialised.(verified it through console)

### CODE REVIEW

File: frontend/src/components/ChatInterface.tsx [hunk 1]
```diff
@@ -44,6 +44,7 @@ function ChatInterface(): JSX.Element {
 
   const handleSendMessage = () => {
     if (inputMessage.trim() !== "") {
+      console.log(inputMessage);
```
Comment (rbren) [inline frontend/src/components/ChatInterface.tsx:47]:
```suggestion
```

File: frontend/src/components/ChatInterface.tsx [hunk 2]
```diff
@@ -55,17 +55,19 @@ function ChatInterface(): JSX.Element {
       <div className="input-container">
         <input
           type="text"
-          value={inputMessage}
```
Comment (rbren) [inline frontend/src/components/ChatInterface.tsx:58]: why did we lose this? seems like it drops two-way binding?

**Comment by Yashwanth-Chandrakumar** [comment]
Hi @rbren yeah the two way binding has been resolved. Can we close this issue and the pull request.

File: frontend/src/components/ChatInterface.tsx (hunk 2 above)
Comment (rbren) [reply frontend/src/components/ChatInterface.tsx:58]: I still see this line deleted--is there a reason for deleting it?

**Comment by rbren** [comment]
@Yashwanth-Chandrakumar I still see that line deleted

**Comment by Yashwanth-Chandrakumar** [comment]
Will look into it and check whether it causes any problem?

**Comment by Yashwanth-Chandrakumar** [comment]
@rbren Checked it throughly there are no issues due to that line. But in order to achieve two way binding we can add that line for sync with state and the UI.

**Comment by rbren** [comment]
Looks like this issue is fixed now. Sorry for the extra work, appreciate you opening a PR!

**Comment by Yashwanth-Chandrakumar** [comment]
Thank you!

**Pull request closed without merging**